storage/mysql/log_storage.go

```diff
@@ -452,7 +453,7 @@ func (t *logTreeTX) QueueLeaves(ctx context.Context, leaves []*trillian.LogLeaf,
 			return nil, fmt.Errorf("queued leaf must have a leaf ID hash of length %d", t.hashSizeBytes)
 		}
 		var err error
-		leaf.QueueTimestamp, err = ptypes.TimestampProto(queueTimestamp)
+		leaf.QueueTimestamp = timestamppb.New(queueTimestamp)
```

Comment:
We probably want CheckValid where there were errors checked before.

Reply:
Wow I really botched this one. Well spotted.